### ISSUE TITLE
update database schema

### ISSUE THREAD

**Issue by fiveddd** [stefanneuhaus/dependencycheck-central-mysql-docker#13]
Hi Stefan,

First, Thanks for this docker images. There was an error when I applied the latest image as `dependency-check` 's central database. The error said we should upgrade the schema of database
![error](https://user-images.githubusercontent.com/28924974/135226841-dfb57d60-181e-43cb-806c-17b24cc1a543.png)

I checked repository of [dependency-check](https://github.com/jeremylong/DependencyCheck/blob/main/core/src/main/resources/data/upgrade_5.1.sql), they have already upgraded database schema. We'd better follow them.

I can send a new pr to fix this problem.

**Comment by stefanneuhaus** [comment]
Hey fiveddd,

Thanks for pointing this out. I´d happily accept your PR. Otherwise I could also provide a fix, but this would probably not happen earlier than the end of next week.

Regards,
Stefan

**Comment by fiveddd** [comment]
Hi,
 Please take a look  (https://github.com/stefanneuhaus/dependencycheck-central-mysql-docker/pull/14).

**Comment by cpfeiffer** [comment]
Hi Stefan, any update on this? The current version of dependency-check does not work with this db anymore. Thanks :-)

**Comment by stefanneuhaus** [comment]
Hey @cpfeiffer and @fiveddd, I'm sorry for the delay. I was wondering on how to proceed as DockerHub discontinued the automatic builds for their free accounts. After a first quick evaluation current plan is to migrate to a GitHub Workflow that builds the Docker image and publishes to DockerHub.

Once this is in place I will provide an updated image that is compatible with the current version of DependencyCheck.

Hopefully I do not run into compatibility issues when building/running/testing the image locally on my new M1 Mac. :see_no_evil:

**Comment by cpfeiffer** [comment]
I see, good luck with your new Mac :smiley: If you need help with the GitHub workflow, please yell. And thanks a lot for maintaining this :heart:

**Comment by stefanneuhaus** [comment]
I fixed the compatibility issue, but did not yet replace the pipeline to publish to DockerHub. Hopefully I will manage to do this in the next couple of days.  

So in the meantime feel free to clone the repository and build your docker image locally, if ypplicable for you.

**Comment by stefanneuhaus** [comment]
I *finally* managed to publish the latest v6.5.3 release to DockerHub. 🎉

Getting the GitHub Actions workflows running took me longer than expected ... 🙈 Thanks for waiting and sorry for the delay!
